### PR TITLE
ALGO-852 Python CLI doesn't list data from connectors

### PR DESCRIPTION
**Pull request by kennydaniel**
CLI was making hard assumptions that data urls start with data:// but that's not always the case (eg- dropbox://, s3://, etc)

### CODE REVIEW

File: Algorithmia/CLI.py [hunk 1]
```diff
@@ -161,75 +161,71 @@ def rm(self, path, client):
...
 
         return listing
 
     # algo cat <file>
     def cat(self, path, client):
         result = ""
         for f in path:
-            if('data://' in f):
-                if(f[-1] == '*'):
+            if '://' in f and not f.startswith("http"):
```
Comment (zeryx) [inline Algorithmia/CLI.py:222]: honestly I'm in favor of this structure over the regex commands we were using in the past. This is much more readable and achieves roughly the same goal.

**Review by zeryx — APPROVED** [review]
LGTM :+1:

**Comment by aslisabanci** [comment]
Hey team, I suggest having a few more tests covering the example inputs Kenny mentioned in the description of the PR.

**Review by John-Bragg — APPROVED** [review]

**Comment by kennydaniel** [comment]
@aslisabanci there are some tests for the CLI stuff, they just didn't cover the s3:// prefix. I'll look at adding additional tests, but would like to get this merged asap since its limiting the use of the CLI.

**Pull request merged**